packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -604,17 +604,75 @@ abstract class BoxScrollView extends ScrollView {
 /// padding. To avoid this behavior, override with a zero [padding] property.
 ///
 /// {@tool sample}
+/// This example uses the default constructor for [ListView] which takes an
+/// explicit [List<Widget] of children. This [ListView]'s children are made up
+/// of [Container]s with [Text].
 ///
-/// An infinite list of children:
+/// ```dart
+/// ListView(
+///   padding: EdgeInsets.all(8.0),
+///   children: <Widget>[
+///     Container(
+///       height: 50,
+///       color: Colors.amber[600],
+///       child: Center( child: Text('Entry A')),
```

Comment:
Center(child: Text('Entry A'))

Note also: the regular Flutter analyzer rules will complain about missing consts in cases like this. Should probably weed those out since devs will copy/paste this code.